components/search/SearchInterface.vue

```diff
@@ -295,7 +302,34 @@
           }
         }
         return updated;
+      },
+      clearFilters() {
+        let qf = this.$route.query.qf;
```

Comment:
this doesn't work when only the REUSABILITY facet is set or only one single other facet.
When it's only reusability it'll be null and when it's a single facet it's a string and won't have a .filter method.
Either use `qf = this.$route.query.qf || [];` and a check that it's an array, or perhaps do: `let qf = [].concat(this.$route.query.qf);` and then remove any nil values.